examples/performance.rs

```diff
@@ -29,10 +31,50 @@ fn performance_with_parameter(param1: String, param2: u32) -> String {
     return format!("Waited {duration:?} for param {param1} - {param2}");
 }
 
+#[get("/performance/skip")]
+fn performance_skipped() -> String {
+    let duration = Duration::from_millis(100);
+    thread::sleep(duration);
+    return format!("Waited {duration:?}\nTransaction will be dropped");
+}
+
+#[get("/performance/rng")]
+fn performance_rng() -> String {
+    let duration = Duration::from_millis(100);
+    thread::sleep(duration);
+    return format!("Waited {duration:?}\nTransaction MIGHT be dropped");
+}
+
 #[launch]
 fn rocket() -> Rocket<Build> {
-    rocket::build().attach(RocketSentry::fairing()).mount(
+    let rocket_instance = rocket::build();
+    let default_rate = rocket_instance
```

Comment:
Could use some explanation:

```suggestion
    // Get the default configured sample rate from `Rocket.toml`
    let default_rate = rocket_instance
```